### PR TITLE
Boiler Plate React Frontend

### PR DESCRIPTION
**Pull request by krachwal**
This is the initial commit for a bare bones React app that loads a web form. There are Cypress end to end tests in place. This is just a standalone frontend.

### CODE REVIEW

**Review by davidbrownell — APPROVED** [review]
I don't know anything about React, but this looks like code to me. Ship it!

**Review by ketanbj — COMMENTED** [review]
npm start output:
Screenshot 2023-11-15 at 9 17 09 AM

I couldn't run  tests using`npx cypress` command: see screenshot 
Screenshot 2023-11-15 at 9 38 13 AM

File: react-frontend/src/components/Form.tsx
[the remark this answers is not in the document]
Comment (krachwal) [reply react-frontend/src/components/Form.tsx:1]: nothing is supposed to happen 😄

File: react-frontend/README.md
[the remark this answers is not in the document]
Comment (krachwal) [reply react-frontend/README.md:1]: will do

**Pull request merged**